Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/CameraFollow.cs

```diff
@@ -4,22 +4,28 @@ namespace Unity.MLAgentsExamples
 {
     public class CameraFollow : MonoBehaviour
     {
-        public Transform target;
-        Vector3 m_Offset;
+        [Tooltip("The target to follow")] public Transform target;
+
+        [Tooltip("The time it takes to move to the new position")]
+        public float smoothingTime; //The time it takes to move to the new position
+
+        private Vector3 m_Offset;
+        private Vector3 m_CamVelocity; //Camera's velocity (used by SmoothDamp)
 
         // Use this for initialization
         void Start()
         {
             m_Offset = gameObject.transform.position - target.position;
         }
 
-        // Update is called once per frame
-        void Update()
+        void FixedUpdate()
```

Comment:
should the camera movement be moved to `LateUpdate`? https://docs.unity3d.com/ScriptReference/MonoBehaviour.LateUpdate.html

Reply:
To ensure that the camera is updating to the most recently updated position of the target